packages/cli/src/__tests__/e2e/helpers/testCliOutput.ts

```diff
@@ -19,6 +19,12 @@ export const testCliOutput = (
     )
   );
 
+  if (expected.exitCode && exitCode !== expected.exitCode) {
```

Comment:
I think the proper condition here would be `if (exitCode !== expected.exitCode)`, no need for the `else if` below if you use this.

Reply:
fixed